build/lib/extensions.ts

```diff
@@ -247,6 +247,7 @@ const externalExtensions = [
 	'arc',
 	'asde-deployment',
 	'azdata',
+	'azurehybridtoolkit',
```

Comment:
Probably want to also add this to recommended extensions : https://github.com/microsoft/azuredatastudio/blob/main/product.json#L42